tests/integration/targets/mso_schema_site_anp_epg_useg_attribute/tasks/main.yml

```diff
@@ -0,0 +1,812 @@
+# Test code for the MSO modules
+# Copyright: (c) 2023, Sabari Jaganathan (@sajagana) <sajagana@cisco.com>
+#
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI MultiSite host, username and password
+  fail:
+    msg: "Please define the following variables: mso_hostname, mso_username and mso_password."
+  when: mso_hostname is not defined or mso_username is not defined or mso_password is not defined
+
+# CLEAN ENVIRONMENT
+- name: Set vars
+  set_fact:
+    mso_info: &mso_info
+      host: "{{ mso_hostname }}"
+      username: "{{ mso_username }}"
+      password: "{{ mso_password }}"
+      validate_certs: "{{ mso_validate_certs | default(false) }}"
+      use_ssl: "{{ mso_use_ssl | default(true) }}"
+      use_proxy: "{{ mso_use_proxy | default(true) }}"
+      output_level: '{{ mso_output_level | default("debug") }}'
+    ignore_errors: true
+
+# Setup Part
```

Comment:
Suggestion:

If we can use same ansible tenant, schema and template name, then cleanup will be redueced
ex:
change -> schema: ans_test_schema -> 
  - { schema: ansible_test } 
  
  For cloud:
    - { schema: aws_ansible_test }
    - { schema: azure_ansible_test }
    
    Also use existing tenant or create tenant with name -> ansible_test.
    
    If applicable in all places.